scripts/set_tradable_tokens.js

```diff
@@ -0,0 +1,39 @@
+const fs = require("fs");
+const MultiSig = require("../build/MultiSigWallet");
+const TokenPriceRegistry = require("../build/TokenPriceRegistry");
+
+const DeployManager = require("../utils/deploy-manager.js");
+const MultisigExecutor = require("../utils/multisigexecutor.js");
+
+async function main() {
+  // Read Command Line Arguments
+  let idx = process.argv.indexOf("--network");
+  const network = process.argv[idx + 1];
+
+  idx = process.argv.indexOf("--input");
+  const input = process.argv[idx + 1];
+
+  // Setup deployer
+  const manager = new DeployManager(network);
+  await manager.setup();
+  const { configurator } = manager;
+  const { deployer } = manager;
+  const deploymentWallet = deployer.signer;
+  const { config } = configurator;
+
+  const TokenPriceStorageWrapper = await deployer.wrapDeployedContract(TokenPriceRegistry, config.modules.TokenPriceRegistry);
+
+  const tokens = JSON.parse(fs.readFileSync(input, "utf8"));
+  const tradable = Array(tokens.length).fill(true);
+
+  // const res = await TokenPriceStorageWrapper.getTradableForTokenList(tokens);
```

Comment:
We should check the `tradable` flag is not set on any of the submitted tokens as the contract reverts when a manager disables a tradable token.